pipelines/pipeline_upgrade_nightly.yml

```diff
@@ -0,0 +1,72 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-katello-3.6-3.7
+    forklift_boxes:
+      pipeline-upgrade-centos7:
+        box: centos7
+        memory: 6144
+  roles:
+    - forklift
+
+- hosts: pipeline-upgrade-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 4
+    katello_repositories_version: 3.6
+    foreman_repositories_version: 1.17
+    foreman_installer_skip_installer: true
```

Comment:
This looks redundant because you don't include the `foreman_installer` role. Since it's just a single system you can however. There's no benefit to splitting it here. We do that on multiple hosts so we can parallelize the base OS install.

Reply:
Gotcha, thanks for the comment!